plugin/arcconf/arcconf.py

```diff
@@ -722,7 +724,7 @@ def volume_delete(self, volume, flags=0):
                 # TODO (Raghavendra) Need to find the scenarios when this can
                 # occur. If volume is detected correctly, but deletion of
                 # volume fails due to arcconf delete command failure.
-                if array_num == ld_info[ld]['logicalDriveID']:
+                if ld_id == ld_info[ld]['logicalDriveID']:
```

Comment:
@cathay4t Is it your continued opinion that this change should be omitted and leave the code as it previously was?

Reply:
array_num does not make sense in arcconf terms as an "array" in arcconf is equivalent to "pool" or "storage pool" as in SNIA terms.

I had changed it to "vol_id" and as @cathay4t mentioned it might confuse with lsmcli volume ID.
So, I think "ld_id" should do good here.